src/rofl/common/ctimer.hpp

```diff
@@ -66,8 +66,8 @@ class ctimer {
 	 *
 	 */
 	ctimer(
-			uint32_t timer_id,
-			const ctimespec& tspec) :
+			uint32_t timer_id = 0,
+			const ctimespec& tspec = ::timespec{0,0} ) :
```

Comment:
Please consider using the alternative constructor: `const ctimespec& tspec(::timespec {0,0});`